packages/next-international/src/common/create-t.ts

```diff
@@ -27,6 +27,11 @@ export function createT<Locale extends BaseLocale, Scope extends Scopes<Locale>
       .filter(key => key.includes('#'))
       .map(key => key.split('#')[0]),
   );
+
+  function getPluralKey(count: number) {
+    if (count === 0) return 'zero';
+    return pluralRules.select(count);
+  }
   const pluralRules = new Intl.PluralRules(context.locale);
```

Comment:
Could you move this `pluralRules` variable before the new `getPluralKey` function? We try to keep things declared before they are used.

Reply:
Sure